input/fsh/CancerGenomics.fsh

```diff
@@ -103,11 +103,6 @@ Conformance statement:
 Observation resources associated with an mCODE patient with an Observation.code in the value set TumorMarkerTestVS MUST conform to this profile. Beyond this requirement, a producer of resources SHOULD ensure that any resource instance associated with an mCODE patient that would reasonably be expected to conform to this profile SHOULD be published in this form, for example, when employing a code that extends the TumorMarkerTestVS value set. Any resource intended to conform to this profile SHOULD populate meta.profile accordingly."
 
 * status and code and subject and effective[x] and value[x] MS
-* bodySite 0..0
```

Comment:
I'm not sure we should constrain it in this manner. It might be that a tumor not located at the primary site is responding or growing, and from the practitioner's point of view, represents the "reason" for the indicated disease status. So I would leave it to the discretion of the clinician.

Reply:
Good point, I agree.